resources/monitoring/values.yaml

```diff
@@ -685,7 +689,15 @@ grafana:
 ##
 kubeApiServer:
   enabled: true
-  tlsConfig:
+  scheme: https
+
+    ## tlsConfig: TLS configuration to use when scraping the endpoint. For example if using istio mTLS.
+    ## Of type: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#tlsconfig
+  tlsConfig: 
```

Comment:
Does this really work? Have you checked the apiserver metrics?

Reply:
Those certs are not used. Already checked it and works as expected